fishtest/fishtest/templates/tests_run.mak

```diff
@@ -186,10 +186,10 @@
     <div class="flex-row stop_rule sprt">
       <label class="field-label leftmost stop_rule sprt">SPRT bounds</label>
       <select name="bounds" class="stop_rule sprt" style="width: 246px">
-        <option value="standard STC">Standard STC {-0.5, 1.5}</option>
-        <option value="standard LTC">Standard LTC {0.25, 1.75}</option>
-        <option value="regression">Non-regression {-1.5, 0.5}</option>
-        <option value="simplification">Simplification {-1.5, 0.5}</option>
+        <option value="standard STC">Standard STC {-0.25, 1.25}</option>
+        <option value="standard LTC">Standard LTC {0.25, 1.25}</option>
+        <option value="regression STC">Non-regression STC {-1.25, 0.25}</option>
+        <option value="regression LTC">Non-regression LTC {-0.75, 0.25}</option>
         <option value="custom" ${is_rerun and 'selected'}>Custom bounds...</option>
       </select>
 
```

Comment:
You will also need to update the default values (-0.5, 1.5) of the fields below.

Reply:
Like this ?
```diff
diff --git a/fishtest/fishtest/templates/tests_run.mak b/fishtest/fishtest/templates/tests_run.mak
index f946b4d..5c8c085 100644
--- a/fishtest/fishtest/templates/tests_run.mak
+++ b/fishtest/fishtest/templates/tests_run.mak
@@ -197,14 +197,14 @@
              style="${args.get('sprt') or 'display: none'}">SPRT Elo0</label>
       <input type="number" step="0.05" name="sprt_elo0"
              class="sprt custom_bounds no-arrows"
-             value="${args.get('sprt', {'elo0': -0.5})['elo0']}"
+             value="${args.get('sprt', {'elo0': -0.25})['elo0']}"
              style="width: 90px; ${args.get('sprt') or 'display: none'}" />
 
       <label class="field-label sprt custom_bounds rightmost"
              style="${args.get('sprt') or 'display: none'}">SPRT Elo1</label>
       <input type="number" step="0.05" name="sprt_elo1"
              class="sprt custom_bounds no-arrows"
-             value="${args.get('sprt', {'elo1': 1.5})['elo1']}"
+             value="${args.get('sprt', {'elo1': 1.25})['elo1']}"
              style="width: 90px; ${args.get('sprt') or 'display: none'}" />
     </div>
 

```

Reply:
Yes, that should be all.